apps/heureka/src/components/filters/FilterPills.jsx

```diff
@@ -5,17 +5,16 @@
 
 import React from "react"
 import { Pill, Stack } from "@cloudoperators/juno-ui-components"
-import { useActiveFilters, useFilterActions } from "../../hooks/useAppStore"
+import { useFilterActions } from "../../hooks/useAppStore"
 import { humanizeString } from "../../lib/utils"
 
-const FilterPills = ({ entityName }) => {
-  const activeFilters = useActiveFilters(entityName)
+const FilterPills = ({ entityName, activeFilters }) => {
```

Comment:
Do you still use `entityName `? I think the var can be removed or?

Reply:
Yes, I do. Since I share the filterActions, I use entityName to update the filter-related properties in the filter slice corresponding to the respective entityName.